lib/assert.js

```diff
@@ -130,6 +130,60 @@ function getBuffer(fd, assertLine) {
   return buffers;
 }
 
+function getErrMessage(call) {
+  const filename = call.getFileName();
+
+  // Skip Node.js modules!
+  if (!filename.includes('/') && !filename.includes('\\')) {
+    return;
+  }
```

Comment:
would it be better to check explicitly against `process.binding('natives')` here?

Reply:
I switched to use that instead. In general: it is possible to show the native code as well, I just think it is not the right thing to do because it might confuse someone getting a error message of e.g. `Reflect.apply(fn, this, foobar)`.